src/test/unit/dashboard_test.test.js

```diff
@@ -0,0 +1,20 @@
+// This is a unit test that takes in the created class (in the file this is just for context, 
```

Comment:
This is a really good unit test. It's unfortunate that it is not testing actual application. It is fine for this assignment.